homeassistant/components/switch/pilight.py

```diff
@@ -0,0 +1,216 @@
+"""
+
+Support for Pilight Switches.
+"""
```

Comment:
The documentation lives at https://home-assistant.io. Usually we remove the configuration details from the header after the merge but for a start it's useful to have at least an example entry if people want to test the PR.